src/core/api/index.js

```diff
@@ -107,12 +108,28 @@ export function callApi({
 }
 
 export function search({ api, page, auth = false, filters = {} }) {
+  const _filters = { ...filters };
+  if (!_filters.clientApp && api.clientApp) {
```

Comment:
I think it would be nice to log a debug message here in case something surprising happens